openmaps_auth/templates/base.html

```diff
@@ -1,6 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
+<script>
+  function addtext(e) {
+    document.querySelector("#default_title").innerHTML = e.getAttribute("data-name")
+    document.querySelector("#featured_application_description").innerHTML = e.getAttribute("data-description")
+  }
+  function removetext(e) {
+    document.querySelector("#featured_application_description").innerHTML = ""
+    document.querySelector("#default_title").innerHTML = document.querySelector("#default_title").getAttribute("data-name")
+  }
+</script>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
```

Comment:
I don't want to external requests, but serving static assets then requires us to collect and serve them, which complicates deployment.

Reply:
Maybe there is a way that I can put svg right into the html and then the different ui elements can reference them, so then there is no need either for an external source or a static file that's served

Reply:
Postscript: not for this release, but using [`whitenoise`](https://whitenoise.evans.io/en/stable/index.html) package will allow us to have application static files served via WSGI instead of Nginx.